shared/actions/profile.js

```diff
@@ -652,16 +635,16 @@ function * generatePgpSaga (): SagaGenerator<any, any> {
     const {shouldStoreKeyOnServer} = finishedAction.payload
 
     // $ForceType
-    const {payload: {response}}: IncomingShouldPush = yield take(generatePgpKeyChan, 'keybase.1.pgpUi.shouldPushPrivate')
+    const {response} = yield takeFromChannelMap(generatePgpKeyChanMap, 'keybase.1.pgpUi.shouldPushPrivate')
```

Comment:
:sheep: a method-based style like `generatePgpKeyChanMap.take()` might be cleaner here.